docs/src/windows.md

```diff
@@ -0,0 +1,106 @@
+# Install and contribute to Erdapfel on Windows
+
+## Windows 7 / 8
+
+- Install React Dev Tools in your browser ([chrome](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi) / [Firefox](https://addons.mozilla.org/fr/firefox/addon/react-devtools/))
```

Comment:
I think we should keep the doc straightforward, and limit it to installing and running the project on Windows.

As React Dev tools aren't related to this topic (and aren't even mandatory to develop), we shouldn't mention them here.
It could be added in a `Recommended tools` section, in another, platform-agnostic doc related to contribution.